touchforms/formplayer/views.py

```diff
@@ -228,14 +228,14 @@ def player_proxy(request):
                                  content_type="text/json", auth=DjangoAuth(auth_cookie))
 
         _track_session(request, json.loads(data), json.loads(response))
-        return HttpResponse(response)
+        return HttpResponse(response, content_type='application/json')
```

Comment:
this didn't fix it, even though [some people suggested it might](http://stackoverflow.com/a/249758/8207). Still seems like a good thing to do.